bin/load_db_scxa_marker_genes.sh

```diff
@@ -83,7 +83,7 @@ if [[ -z ${NUMBER_MGENES_FILES+x} || $NUMBER_MGENES_FILES -gt 0 ]]; then
   # Load data
   echo "Marker genes: Loading data for $EXP_ID..."
   printf "\copy scxa_marker_genes (experiment_accession, gene_id, k, cluster_id, marker_probability) FROM '%s' WITH (DELIMITER ',');" $EXPERIMENT_MGENES_PATH/mgenesDataToLoad.csv | \
-    psql $dbConnection
+    psql -v ON_ERROR_STOP=1 $dbConnection
```

Comment:
Can we add something here to delete the table (rollback) if the above operation has an error code? It would be essentially repeating line 44 and then exiting with and error code. We might need to check for any set -e higher up.